src/interface/term.ml

```diff
@@ -931,6 +940,46 @@ module type Ae_Bitv = sig
 
 end
 
+module type Ae_Float_Float = sig
```

Comment:
Is this module type necessary ? Why not just manually inline it at its only use site ? Or do you envision it being used elsewhere ?